docs/sections/workflows/workchain.md

```diff
@@ -613,101 +621,149 @@ Out[3]:
+### Submit the work chain
+
+When _submitting_ work chains to the AiiDA daemon, it's important that it knows where to find the work chain module, i.e. the `.py` file that contains the work chain code.
+To do this, we need to add the directory that contains this file to the `PYTHONPATH`.
+Make sure you are in the directory that contains the `addcalcjobworkchain.py` file and execute:
 
 ```{code-block} console
-$ echo "export PYTHONPATH=\$PYTHONPATH:$PWD" >> /home/max/.virtualenvs/aiida/bin/activate
+$ echo "export PYTHONPATH=\$PYTHONPATH:$PWD" >> **TODO FIX THIS PATH**
 $ verdi daemon restart --reset
 ```
 
-Check if you have setup the `add` code:
+:::{important}
+The code snippet above only works for the `base` environment on the AiiDAlab JupyterHub cluster.
+If you are working in a different environment, or on your local machine, you have to replace the path of the `activate` script to that of your environment.
+:::
+
+Also double check that you have set up the `add` code used in the `ref`{AiiDA basics module <fundamentals-basics-calcjobs>}:
 
 ```{code-block} console
 $ verdi code list
-# List of configured codes:
-# (use 'verdi code show CODEID' to see the details)
+...
 * pk 1912 - add@localhost
-
 ```
 
-Setup the computer and code if needed:
+If not, you can set it up with the instructions in the dropdown below.
+
+:::{dropdown} Set up the `add` code
+
+Setting up the `add` code on the `localhost` computer can be done with the following command:
 
 ```{code-block} console
-$ verdi computer setup --config computer.yml
-$ verdi computer configure local localhost
-$ verdi code setup --config code_add.yml
+$ verdi code setup -L add --on-computer --computer=tutor -P arithmetic.add --remote-abs-path=/bin/bash -n
 ```
 
-In the `verdi shell`, run:
-```{code-block} console
+See the {ref}`calculation jobs section <fundamentals-basics-calcjobs>` in the AiiDA basics module for mode details.
+
+:::
+
+Then open a `verdi shell` and import the `submit()` function and work chain:
+
+```{code-block} ipython
 In [1]: from aiida.engine import submit
-In [2]: from multiply_add_workchain_calcjob import MultiplyAddWorkChain
-In [3]: add_code = load_code(label='add@localhost')
-In [4]: inputs = {'x': Int(1), 'y': Int(2), 'z': Int(3), 'code': add_code}
-In [5]: workchain_node = submit(MultiplyAddWorkChain, **inputs)
+   ...: from addcalcjobworkchain import AddCalcjobWorkChain
 ```
 
-Now, go and check the last processes, the status of the `MultiplyAddWorkChain`, and show some details about the inputs and outputs.
+We'll also need the `add` code set up on the `localhost`.
+Load it using its label:
 
-## Cleaning up
+```{code-block} ipython
+In [2]: add_code = load_code(label='add@localhost')
+```
 
-**TODO: Add example on how to kill/delete a process to clean up erroneous `AddWorkChain` run.**
+Then we can submit the work chain:
+
+```{code-block} ipython
+In [3]: workchain_node = submit(AddWorkChain, x=Int(1), y=Int(2), code=add_code)
```

Comment:
I think that here we should submit `AddCalcjobWorkChain` instead of the `AddWorkChain`.